cmd/cloud/deployment_inspect_test.go

```diff
@@ -105,7 +106,8 @@ var (
 	}
 	mockCoreDeploymentResponse = []astrocore.Deployment{
 		{
-			Status: "HEALTHY",
+			Status:           "HEALTHY",
+			WorkloadIdentity: &workloadIdentity,
```

Comment:
is this test actually validating anything about the deployment values returned? it probably isn't, since this issue wasn't caught previously. should we make it validate that the deployment has the correct attributes?

Reply:
oh its not...it was throwing an nil exception since we didnt have the check..let me update this

Reply:
it looks like we're still not validating any values in the test?